owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -334,48 +353,35 @@ public void onClick(View v) {
                         getResources().getString(R.string.app_name)));
                 final BottomSheetBehavior uploadBottomSheetBehavior =
                         BottomSheetBehavior.from((View) uploadBottomSheet.getParent());
-                dialog.setOnShowListener(new DialogInterface.OnShowListener() {
-                    @Override
-                    public void onShow(DialogInterface dialog) {
-                        uploadBottomSheetBehavior.setPeekHeight(uploadBottomSheet.getMeasuredHeight());
-                    }
-                });
+                dialog.setOnShowListener(dialog1 ->
```

Comment:
dialog1 is not a good variable name 😉

Reply:
Auto generated, i would call it `_` if it was a kotlin file